extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -154,7 +150,7 @@ require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTrack
 			},
 			function () {
 				return {
-					searchType: getCurrentScope()
+					searchType: currentScope
 				};
```

Comment:
I meant that this parameter doesn't need to be a function, `resultClickTrackerFactory` can be modified.